forch/faucet_state_collector.py

```diff
@@ -181,15 +181,18 @@ def _update_packet_count_states(self, vlan_counts, interval):
             threshold = self._packet_per_sec_thresholds.get(vlan_id)
 
             if not threshold:
-                continue
-
-            if rate > threshold:
+                rate_state = State.unknown
+            elif rate > threshold:
                 LOGGER.error(
                     'Packet per sec for vlan %d is greater than threshold %d: %.2f',
                     vlan_id, threshold, rate)
-                last_vlan_map[PACKET_RATE_STATE] = State.broken
+                rate_state = State.broken
             else:
-                last_vlan_map[PACKET_RATE_STATE] = State.healthy
+                rate_state = State.healthy
+
+            last_vlan_map[PACKET_RATE_STATE] = rate_state
+            self._forch_metrics.update_var(
+                'vlan_packet_rate_state', rate_state, labels=[vlan_id])
```

Comment:
should expose actual counts (could also do state too, but we want the raw numbers exposed because they can be used to make pretty graphs)

Reply:
Expose the raw counts now. PTAL..